Tests/kaas/k8s-default-storage-class/k8s-default-storage-class-check.py

```diff
@@ -180,28 +184,139 @@ def check_default_persistentvolumeclaim_readwriteonce(k8s_api_instance, storage_
                     "access mode 'ReadWriteOnce' is not supported",
                     return_code=42,
                 )
+    return 0
 
-    # 4. Delete resources used for testing
-    logger.debug(f"delete pod:{pod_name}")
-    api_response = k8s_api_instance.delete_namespaced_pod(pod_name, namespace)
-    logger.debug(f"delete pvc:{pvc_name}")
-    api_response = k8s_api_instance.delete_namespaced_persistent_volume_claim(
-        pvc_name, namespace
-    )
 
+def check_csi_provider(k8s_core_api, allowed_csi_prov=ALLOWED_CSI_PROV):
```

Comment:
What is this good for, and how was the list `ALLOWED_CSI_PROV` determined? I can't see any mention of this in the standard nor the testing notes. A comment in the code is required, but also some explanation in the testing notes.

Reply:
Okay, but this doesn't address any of my points.